apps/server/src/modules/tool/external-tool/loggable/external-tool-logo-fetch-failed-loggable-exception.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpStatus } from '@nestjs/common';
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+import { BusinessError } from '@shared/common';
+
+export class ExternalToolLogoFetchFailedLoggableException extends BusinessError implements Loggable {
+	constructor(
+		private readonly logoUrl: string,
+		private readonly httpStatus?: HttpStatus,
+		private readonly externalToolId?: string
```

Comment:
Are you using this?

Reply:
it is used for the data section of the loggable. The externalToolId is not yet used, but it could make sense to use this in the future ^^